conf/test/manual.csv

```diff
@@ -94,7 +94,8 @@ TestStep,,,,2,Check existing file using command getfattr -d -m . path_to_file to
 TestStep,,,,3,"use ""swupd verify"" to make sure current flashed image is in good status for swupdate",0 file shoud mismatch,,,,,,,
 TestStep,,,,4,"apply update task using ""swupd_update -v -c"" and make sure update complete successfully, this will redownload the update repo",the update should complete successfully and corresponding files are changed,,,,,,,
 TestStep,,,,5,"do ""swupd verify"" to check system is same as official image",check system should be in good status,,,,,,,
-TestStep,,,,6,repeat step 1),fail to modify existing files in system.,,,,,,,
+TestStep,,,,6,"if UEFI partition get updated(check commits messages), do ""fdisk -l mmcblk0p1"" before update and after reboot to check UEFI partition switched to new one.","EFI systems should change to new one, refer IOTOS-1358 for comparison",,,,,,,
```

Comment:
okay. Let me update to use sgdisk. By the way, how to check the new.efi is used in booting? maybe there will be no explicit info during booting to identify new efi is used.